pkgs/development/libraries/mesa/default.nix

```diff
@@ -5,7 +5,7 @@
 , libelf, libvdpau, python3Packages
 , libglvnd
 , enableRadv ? true
-, galliumDrivers ? ["auto"]
+, galliumDrivers ? ["auto" "iris" ]
```

Comment:
"iris" should just be built on linux

Reply:
https://github.com/NixOS/nixpkgs/pull/65017#issuecomment-512896254